flexget/plugins/search/1337x.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import unicode_literals, division, absolute_import
+from future.moves.urllib.parse import quote
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+
+from flexget import plugin, db_schema
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import TimedLimiter, RequestException
+from flexget.utils.requests import Session as RequestSession
+from flexget.utils.soup import get_soup
+from flexget.utils.search import torrent_availability
+
+log = logging.getLogger('1337x')
+Base = db_schema.versioned_base('1337x', 0)
+
+requests = RequestSession()
+requests.add_domain_limiter(TimedLimiter('1337x.to', '5 seconds'))  # TODO find out if they want a delay
+
+class _1337x(object):
```

Comment:
because the class name starts with a number so without the underscore it was throwing a syntax error.  The other plugins are all the name of the torrent site so just did this as a workaround.